docs/migrationguide/9_0.md

```diff
@@ -65,7 +65,11 @@
 ## Cache
   * HazelCast now requires version 4.X see: (https://github.com/primefaces/primefaces/issues/5989)
 
-## Exporter
+## (Data)Exporter
```

Comment:
I think its safe to remove the parethesis since DataExporter is the name of the component.

Reply:
Done